code/backend/batch/utilities/helpers/env_helper.py

```diff
@@ -4,6 +4,7 @@
 from dotenv import load_dotenv
 from azure.identity import DefaultAzureCredential, get_bearer_token_provider
 from azure.keyvault.secrets import SecretClient
+from backend.batch.utilities.helpers.config.conversation_flow import ConversationFlow
```

Comment:
Because of the weird project structure, using an absolute import here breaks the Admin app and Azure Function when running in Azure

I think that using a relative import should fix it